src/main.cpp

```diff
@@ -255,10 +254,10 @@ void loop()
     {
       converter.readRegistryValues(buff); //process all values from the register
       updateValues(registryIDs[i]);       //send them in mqtt
-      waitLoop(500);//wait .5sec between registries
+      //waitLoop(500);//wait .5sec between registries
```

Comment:
I'm not sure whether and why a waitLoop here is necessary.

Reply:
That was maybe an unnecessary precaution... OK to remove it.